tests/admin_changelist/test_actions.py

```diff
@@ -0,0 +1,91 @@
+from django.contrib import admin
```

Comment:
I would put the tests in the directory where the other test is reverted but I'm open to hearing why you think they belong here.

Reply:
Yes, having them as part of `modeladmin` would work too. 

My thinking was that A) bar the reverted test, there were no existing tests on _actions_ (so no clear _it must go here_) and B) that _actions_ are part of/related to `ChangeList`s, rather than `ModelAdmin` in general, so the new tests could live in the specialised `admin_changelist` module. 

Happy to move them if you prefer.

Reply:
My thinking is that `tests/modeladmin` tests `ModelAdmin` methods. I guess `admin_changelist` is supposed to be for `admin.views.ChangeList`.

Reply:
Yeah, I can see it either way. I'll move them. 👍